amy/fiscal/tests/test_filters.py

```diff
@@ -0,0 +1,454 @@
+from datetime import date
+
+from django.db.models import Count, F, Q
+from django.db.models.functions import Coalesce
+from django.test import TestCase
+
+from fiscal.filters import MembershipFilter, MembershipTrainingsFilter
+from fiscal.models import Membership
+from workshops.models import Event, Member, MemberRole, Organization, Person, Role, Task
+
+
+class TestMembershipFilter(TestCase):
+    @classmethod
+    def setUpClass(cls) -> None:
```

Comment:
Is there a particular reason why you chose `setUpClass` and `tearDownClass` class methods?

[The docs](https://docs.python.org/3/library/unittest.html#class-and-module-fixtures) say it should be used with caution:

> Note that shared fixtures do not play well with [potential] features like test parallelization and they break test isolation. They should be used with care.

In all other cases we used `setUp()` and `tearDown` methods.

Reply:
I am a little afraid about parallelization of these tests (I use it often with `make fast_test` or `python manage.py test --parallel` commands).

Reply:
Habit from a past project, and a desire to keep the number of database calls down. Happy to change to `setUp/tearDown`. (and actually learned about the helper setup methods in TestBase since I wrote this, so will look at using those too)